Server/shop/shop_database/src/main/java/com/cashmanager/server/shop_database/config/FlywayConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package com.cashmanager.server.shop_database.config;
+
+
+import jakarta.annotation.PostConstruct;
+import org.flywaydb.core.Flyway;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Profile("dev")
+@Configuration
+public class FlywayConfiguration {
```

Comment:
Explanation for others,
Doc for Flyway and how to use it : https://www.baeldung.com/database-migrations-with-flyway